jobs/gtfs-schedule-validator/gtfs_schedule_validator.py

```diff
@@ -172,24 +189,28 @@ def download_and_validate_extract(
             pbar=pbar,
         )
         fs.get_file(extract.path, zip_path)
-        report, system_errors = execute_schedule_validator(
+
+        report, system_errors, validator_version = execute_schedule_validator(
+            extract_ts=extract.ts,
             zip_path=Path(zip_path),
             output_dir=tmp_dir,
             pbar=pbar,
         )
+
     validation = GTFSScheduleFeedValidation(
         filename=f"validation_notices{JSONL_GZIP_EXTENSION}",
```

Comment:
ok so this is **not** putting the validator version in the filename?

Reply:
oops ty